From 78668f1393f91253cf5ed3ef3cbae3f70523292b Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <akitouni@gnome.org>
Date: Sun, 21 Feb 2021 13:41:09 +0100
Subject: [PATCH] components/linux.bst: compile features needes for live boot
 as modules

There seems to be something wrong when they're compiled in the kernel
---
 elements/components/linux.bst | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/elements/components/linux.bst b/elements/components/linux.bst
index fec68f56..180c9ed8 100644
--- a/elements/components/linux.bst
+++ b/elements/components/linux.bst
@@ -137,10 +137,10 @@ config:
     scripts/config -e SND_HDA_GENERIC
 
     # Required for live boot
-    scripts/config -e SQUASHFS
-    scripts/config -e SQUASHFS_ZLIB
-    scripts/config -e DM_SNAPSHOT
-    scripts/config -e OVERLAY_FS
+    scripts/config -m SQUASHFS
+    scripts/config -m SQUASHFS_ZLIB
+    scripts/config -m DM_SNAPSHOT
+    scripts/config -m OVERLAY_FS
 
     # Some useful drivers when runnig as virtual machine
     scripts/config -m VIRTIO_BALLOON
-- 
2.30.1

